src/components/Tabs/TabList/TabList.scss

```diff
@@ -0,0 +1,94 @@
+@import "../../../styles/themes";
+@import "../../../styles/typography";
+
+.tabs--wrapper {
+  padding-bottom: 20px;
+
+  &.disabled {
+    opacity: 0;
+    height: 0;
+    padding-bottom: 5px;
+    pointer-events: none;
+    ul li a:hover {
```

Comment:
In my opinion, it is better to target classes or attributes rather with native elements selectors can this be changed?